curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b1ee1de7216896b5b4361.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 657b1ee1de7216896b5b4361
+title: Task 38
+challengeType: 22
+dashedName: task-38
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: That's interesting! So, you didn't work on many programming projects during your studies?
+Sophie: No, not really. I played around with coding, but most of my projects involved circuit designs and electrical systems. It was a bit different. 
+-->
+
+# --description--
+
+Listen to the dialogue and fill in the blank correctly. This task is a review. Remember, the simple past tense is used to talk about actions or situations in the past, and the negative form is used to describe things that did not happen.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Sarah: That's interesting! So, you _ work on many programming projects during your studies?`
+`Sophie: No, not really. I _ around with coding, but most of my projects involved circuit designs and electrical systems. It was a bit different.`
```

Comment:
we need to practice these two structures, so I won't delete any lines. Ideally, we could break this challenge in two.

Reply:
```suggestion
`Sarah: That's interesting! So, you _ work on many programming projects during your studies?`

`Sophie: No, not really. I _ around with coding, but most of my projects involved circuit designs and electrical systems. It was a bit different.`
```
In anticipation of https://github.com/freeCodeCamp/freeCodeCamp/pull/52552